apps/meteor/client/components/message/content/ThreadMetricsParticipants.tsx

```diff
@@ -0,0 +1,29 @@
+import { MessageMetricsItem, MessageMetricsItemLabel, MessageMetricsItemAvatarRow } from '@rocket.chat/fuselage';
+import { useTranslation } from '@rocket.chat/ui-contexts';
+import type { ReactElement } from 'react';
+import React from 'react';
+
+import ThreadMetricAvatar from './ThreadMetricAvatar';
+
+type ThreadMetricsParticipantsProps = {
+	participants: Array<string>;
+};
+
+const ThreadMetricsParticipants = ({ participants }: ThreadMetricsParticipantsProps): ReactElement => {
+	const t = useTranslation();
+
+	const participantsLabel = participants.length - 2 > 0 ? `+${participants.length - 2}` : undefined;
```

Comment:
What does `participants.length - 2` mean? Wouldn't it be clearer to have a named variable with the meaning (business speaking) of this?